CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 4.2.0 (2020-07-08)
+
+- Added the ability to disable the autocomplete component via a reactive form. [#101](https://github.com/blackbaud/skyux-lookup/pull/101)
```

Comment:
Is this only for reactive forms? If so, do we need to update the description in the docs?

Also, should `via` should be `on`? I mean, I assume we're talking about disabling autocomplete components ON reactive forms, not using reactive forms to disable autocomplete components, right?